readthedocs/index.rst

```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
 
 .. code-block:: python
 
+   import sys
+   import os
+   os.chdir(sys.path[0])
+   
+   session_name = "name"
+   
+   if f"{session_name}.session" in os.listdir():
+      os.remove(f"{session_name}.session")
```

Comment:
Constantly having to login again is not something we want to teach (and that's a bad way to check if a file exists).

Reply:
please see [this](https://stackoverflow.com/a/60978983/4999991)